### PR TITLE
Add support for available locales

### PR DESCRIPTION
**Pull request by gittymcgit**
## Context

We need to have links that fallback to /gb/ in some locales
### Changes

I added `availableLocales` field, that takes a list of locales in which the page exists. If customer's locale is not in the list, then we should fallback the link to gb.

If the list is not provided for a link, then the link exists in all locales
## Checklist
- [x] I have considered the impact of this change and added a [Change Classification](
https://transferwise.atlassian.net/wiki/spaces/EKB/pages/1401189673/Change+Classifications+and+Expectations) Label (`change:standard`, `change:impactful`, `change:emergency`)
- [x] Change meets or does not compromise the [Baseline Security Requirements](https://transferwise.atlassian.net/wiki/spaces/EKB/pages/434929973/Baseline+Security+Requirements)

### CODE REVIEW

File: items/business-navigation.json [hunk 1]
```diff
@@ -36,15 +36,32 @@
...
+            "pl",
+            "jp",
+            "it",
+            "fr",
+            "es",
+            "de",
+            "us",
+            "hu",
+            "tr",
+            "gb"
```
[the remark this answers is not in the document]
Comment (gittymcgit) [reply items/business-navigation.json:53]: I'm not sure. Feels a bit implicit without it

[the remark this answers is not in the document]
Comment (karanjthakkar) [reply items/business-navigation.json:53]: The doc change you added seems fairly clear that the purpose of that key is to tell which locales won't fallback to `gb`. Hence, my comment.

**Comment by brettyukich** [comment]
These fallbacks may be useful outside of the public-navigation. For example any CTA that links to one of these business pages.

Organic Growth has a similar project in https://github.com/transferwise/product-locale-directory, but it's Java only :( 

Recently we've created https://github.com/transferwise/sensible which currently contains sesible amounts and sensible routes. The sensible repo publishes both a npm package and a java library using the same configuration :)

Perhaps we could consolidate this configuration into the sensible library?

**Comment by danlaush** [comment]
@brettanomyces I guess I would worry that the sensible package would become a grab-bag of config, which doesn't feel quite right to me. It's a good point about other CTAs though 🤔 will bring that point in Slack for discussion.

**Comment by twrdjpalmer** [comment]
I realise it has probably already been considered, but this seems to lend itself to creating a very janky experience for people who are viewing non-gb pages. To go from `/fr` (when in some cases this dictates language, also) to `/gb` seems like a subpar experience.

Can you provide details as to why we're opting for that approach vs not showing the link at all vs creating the market-specific pages that are missing?

**Comment by gittymcgit** [comment]
@twrdjpalmer Yeah, the ideal solution would be to have a page for every locale, but we don't have them yet. This PR brings it back to what it used to be before lienzo-frontend got released. These links used to be replaced by Lienzo backend. 
We're still showing the link, because the product is present in the country, we just don't have a localised page for it

**Comment by brettyukich** [comment]
> Can you provide details as to why we're opting for that approach vs not showing the link at all vs creating the market-specific pages that are missing?

Better to fallback to a actual page than have a link 404'ing. 

But as you suggest better to not how the link at all, to do so you still need to know which pages are supported in which locales.  Maybe the product availability API is a better source of truth for this? (Which repo does it live in by the way?)

**Comment by gittymcgit** [comment]
@brettanomyces the product is still available tho

**Comment by danlaush** [comment]
> product availability API is a better source of truth

That's an interesting approach. It would make the nav dependent on an async request though, which would add some complexity. The internal sidebar has recently moved to that model, but I think landing pages have a different calculus.

**Comment by twrdjpalmer** [comment]
@brettanomyces [Consumer onboarding service](https://github.com/transferwise/consumer-onboarding-service#usage). Don't think we have 1:1 marketing pages:availability though.

@gittymcgit OK, taking this more as a bug fix then 👍 . I do think we should prioritise removing the behaviour in favour of adding the missing content in relevant markets however

**Comment by gittymcgit** [comment]
I agree! Ideally we should have specific pages for all locales where product is available

**Review by karanjthakkar — APPROVED** [review]

**Pull request merged**